region/maridia/outer.json

```diff
@@ -1303,12 +1295,31 @@
                   ]
                 },
                 {
-                  "name": "Suitless Frozen Enemies",
+                  "name": "Suitless Frozen Crab",
                   "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
                     "HiJump",
-                    "canUseFrozenEnemies"
+                    "canTrickyUseFrozenEnemies",
+                    "h_canCrouchJumpDownGrab"
+                  ],
+                  "note": [
+                    "The first crab, coming down, is a bit tricky to freeze fast enough without any other beams or a missile.",
+                    "Prevent it from coming on screen until Samus is in position to hit it accurately."
+                  ]
+                },
+                {
+                  "name": "Suitless Frozen Global Crab",
+                  "notable": false,
+                  "requires": [
+                    "canSuitlessMaridia",
+                    "HiJump",
+                    "canUseFrozenEnemies",
+                    "h_canCrouchJumpDownGrab",
```

Comment:
You can avoid crouchJumpDownGrab by freezing global crab in friendlier places for these two jumps where you'd want it.

Reply:
And springball jumps work in place of HiJump